README.md

```diff
@@ -74,6 +74,64 @@ manager = TransferManager(client, config)
 manager.download_read_set(SEQUENCE_STORE_ID, "<my-read-set-id>")
 ```
 
+## Using the Omics URI Parser
```

Comment:
You could consider merging the URI Parser guide with the Transfer Manager guide.  There's a lot of duplicated example code this way, though it also makes it clear that the Transfer Manager can be used without the URI parser.

Up to you; I'm fine with either approach.

Reply:
Would prefer to keep this separate so that it's clear the 2 tools can be used separately